server/conf/application.conf

```diff
@@ -307,6 +307,7 @@ play.filters {
   enabled += filters.HSTSFilter
   enabled += filters.LoggingFilter
   enabled += filters.ValidAccountFilter
+  enabled += filters.gzip.GzipFilter
```

Comment:
Out of curiosity, why only in prod mode?

When viewing the response in devtools, the response will be un-zipped into its full form.

Reply:
just to reduce complexity and stack trace size when applicable. probably shouldn't be blocking, changed the comment

Reply:
I'm not sure I understand how disabling gzip in dev would reduce complexity. It seems that it would be something we'd have to remember is different between the two. From the browser and server perspective, performing gzip would be more of a network-layer concern and is largely transparent on either side (e.g. even though the content is gzipped, the dev tools still render the un-gzipped content).

Personally, I'd prefer to keep things consistent between the two stacks. Having this enabled for dev also means we'll get more eyes on it should something go wrong. If we disable in dev, it's possible that the issues will only occur in deployments, which seems unfortunate.

Reply:
+1. I don't think that enabling in dev with have any real impact on complexity.  And it's always nice to keep it as similar to prod as possible

Reply:
I have re-enabled it in dev

Reply:
I added [this](https://github.com/seattle-uat/civiform/pull/2612/commits/3fc762ab824da8bc552909374c8dfa60cc6e9f5c) and got this trace

![flat_earth_exception](https://user-images.githubusercontent.com/7295000/171970161-6a8ef766-603a-4cab-8f73-fb504a426eb6.png)

Reply:
WIthout gzip that stack trace looks like

![flat_earth_exception_nogzip](https://user-images.githubusercontent.com/7295000/171970397-2b8e2bef-10ca-41e5-b06e-84f8a673680e.png)